utilities.tex

```diff
@@ -785,28 +792,28 @@
     tagged(const tagged&) = default;
     tagged &operator=(tagged&&) = default;
     tagged &operator=(const tagged&) = default;
-    tagged(Base&&) noexcept(@\seebelow@)
+    constexpr tagged(Base&&) noexcept(@\seebelow@)
       requires MoveConstructible<Base>;
-    tagged(const Base&) noexcept(@\seebelow@)
+    constexpr tagged(const Base&) noexcept(@\seebelow@)
```

Comment:
I was hoping to get away with these two; it's clearly in line with the intent of the change.